src/main/java/ui/enums/HeaderMenuPageTitle.java

```diff
@@ -0,0 +1,35 @@
+package ui.enums;
+
+
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@Getter
+public enum HeaderMenuPageTitle {
+    BOOKS("BOOKS", "Books", "https://demowebshop.tricentis.com/books"),
+    COMPUTERS("COMPUTERS", "Computers", "https://demowebshop.tricentis.com/computers"),
+    ELECTRONICS("ELECTRONICS", "Electronics", "https://demowebshop.tricentis.com/electronics"),
+    APPAREL_AND_SHOES("APPAREL&SHOES", "Apparel & Shoes", "https://demowebshop.tricentis.com/apparel-shoes"),
+    DIGITAL_DOWNLOADS("DIGITAL DOWNLOADS", "Digital downloads", "https://demowebshop.tricentis.com/digital-downloads"),
+    JEWELRY("JEWELRY", "Jewelry", "https://demowebshop.tricentis.com/jewelry"),
+    GIFT_CARDS("GIFT CARDS", "Gift Cards", "https://demowebshop.tricentis.com/gift-cards");
```

Comment:
Do we really need to use the full URL here?
Maybe the better way is to use path like "/books"?
Also we have base url in properties.

Reply:
Done